pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/kubeflow.org/v1/PyTorchJob/customizations.yaml

```diff
@@ -162,55 +164,97 @@ spec:
-                  -- but generate a new condition by PytorchJob finish state.
-                  -- table.insert(aggregatedConditions, c)
-                  if (c.type == "Succeeded" or c.type == "Failed") and c.status == "True" then
-                    isFinished = true
-                    finishedType = c.type
+                  if c.type == "Failed" and c.status == "True" then
+                    hasFailed = true
+                  elseif c.type == "Succeeded" and c.status == "True" then
+                    hasSucceeded = true
+                  end
+                  
+                  if c.type == "Running" then
+                    hasRunningCondition = true
+                  end
+                  
+                  if c.type == "Created" and c.status == "True" then
+                    hasCreated = true
                   end
                 end
               end
 
-              if isFinished then
-                if finishedType == "Succeeded" then
-                  successfulClustersNum = successfulClustersNum + 1
-                elseif finishedType == "Failed" then
-                  table.insert(failedClusters, statusItems[i].clusterName)
-                end
+              -- Count clusters by status
+              if hasFailed then
+                table.insert(failedClusters, statusItems[i].clusterName)
+              elseif hasSucceeded then
+                successfulClustersNum = successfulClustersNum + 1
+              end
+
+              if hasCreated then
+                createdClustersNum = createdClustersNum + 1
               end
 
             end
           end
             
+          -- Set conditions in order: Created -> Running -> Succeeded/Failed (mutually exclusive)
+          -- kubectl get shows the last condition, so order matters
+          
+          -- 1. Created condition (earliest state, can coexist with others)
+          if createdClustersNum > 0 then
+            table.insert(aggregatedConditions, {
+              type = "Created",
+              status = "True",
+              lastUpdateTime = startTime,
+              lastTransitionTime = startTime,
+              reason = "PyTorchJobCreated",
+              message = "PyTorchJob has been created"
+            })
+          end
+          
+          -- 2. Running condition (intermediate state, can coexist with Created)
+          if hasRunningCondition then
+            local hasFinalState = (#failedClusters > 0) or (successfulClustersNum == #statusItems and successfulClustersNum > 0)
+            local runningStatus = "True"
+            if hasFinalState then
+              runningStatus = "False"
+            end
+            table.insert(aggregatedConditions, {
+              type = "Running",
+              status = runningStatus,
+              lastUpdateTime = os.date("!%Y-%m-%dT%H:%M:%SZ"),
+              lastTransitionTime = os.date("!%Y-%m-%dT%H:%M:%SZ"),
+              reason = "PyTorchJobRunning",
```

Comment:
Is the value of reason the same regardless of whether the value of runningStatus is True or False?